Adding initial set of content for "useless box" work shop. (#336)
NuevoFoundation/workshops: content/english/useless-box/build-the-box.md

---
title: "Build the box"
description: ""
draft: true
weight: 3
---

## Let's build the box
![diagram of the parts to build the box, base, lids, sides, end, motor mount and finger](../img/boxPartsDiagram.png)

> 1.	Glue 2 fingers together, to make a thicker finger. (If it�s already thick enough, skip this step)

Reviewer: How thick is "thick enough"? Can you give an exact measurement in cm/inches?

```suggestion
1.	Glue 2 fingers together, to make a thicker finger. (If it's already thick enough, skip this step).
```

Author: this will get updated, as we're taking pictures and will provide more details on the materials

> 2.	Attach the motor to the motor mount
> 3.	Attach the momentary/limit switch 
> 4.	Mount the toggle switch onto the lid
> 5.	Press the finger onto the motor shaft and secure it
> 6.	The rest of the box slides and snaps together.
> 	a.	Fit the motor mount piece to the base, try to get the finger to be centered in the box
> 	b.	Fit the sides
> 	c.	Put one lid on and snap the end. 

Reviewer: ```suggestion
2.	Attach the motor to the motor mount.
3.	Attach the momentary/limit switch.
4.	Mount the toggle switch onto the lid.
5.	Press the finger onto the motor shaft and secure it.
6.	The rest of the box slides and snaps together.
	a.	Fit the motor mount piece to the base, try to get the finger to be centered in the box.
	b.	Fit the sides.
	c.	Put one lid on and snap the end. 
```

> Should look something like this:

Reviewer: ```suggestion
Now, it should look something like this:
```

---
title: "Build the box"
description: ""
draft: true
weight: 3
---

> ## Let�s build the box

Reviewer: ```suggestion
## Let's build the box
```

Reviewer: I think all the quotation marks have been messed up in this PR

![diagram of the parts to build the box, base, lids, sides, end, motor mount and finger](img/boxPartsDiagram.png)

> 1.	Glue 2 fingers together, to make a thicker finger. (If it�s already thick enough, skip this step)

Reviewer: Is it possible to have pictures for assembly of each step?

Author: yes, we'll be taking pictures and updating this page, we wanted to put some initial content in a first PR, so it gives some shape and can be collaborated on.

2.	Attach the motor to the motor mount
3.	Attach the momentary/limit switch 
4.	Mount the toggle switch onto the lid
5.	Press the finger onto the motor shaft and secure it
6.	The rest of the box slides and snaps together.
	a.	Fit the motor mount piece to the base, try to get the finger to be centered in the box
	b.	Fit the sides
	c.	Put one lid on and snap the end. 
Should look something like this:
![4 images showing steps to build the box](img/boxBuildingSteps.png)

Here�s the circuit design:
![diagram of the final circuit](img/circuit.png)
When the limit switch connects and something (for example: you) changes the position of the toggle switch ON, the battery powers the motor, which moves the finger.  
The mechanical finger from inside the box comes out and toggles the switch OFF, so the mechanical finger retracts.

> ### *CONGRATS! You�ve done it. Feel free to play with your box and learn or experiment with more basic circuits.*

Reviewer: ```suggestion
### CONGRATS! You�ve done it. Feel free to play with your box and learn or experiment with more basic circuits.
```

Supporting resources: 

1. http://makezine.com/projects/the-most-useless-machine/
2. http://www.youtube.com/watch?v=aqAUmgE3WyM

---

> ![Nuevo Foundation robot mascot](img/NF_mascot.jpg)

Reviewer: This workshop completed section should be filled in automatically once the workshop is published

- The **motor** is connected to 2 wires (green, blue). 
![motor with 2 wires](../img/motorWith2Wires.jpg)
- The motor should have an acrylic finger attached.
![motor with acrylic finger](../img/motorWithFinger.jpg)
- The top **toggle switch** is connected to 6 wires (yellow, green, 2 blue, red, black) from its 6 terminals labeled in the figure above. 
It should look something like this:
![switch with wires](../img/switchWithWires.jpg)

Here's a broader view of the part that has the motor, finger and limit switch attached:
![view of piece with motor, finger and limit switch with wires attached](../img/motorFingerAndLimitSwitchPiece.jpg)
- A **breadboard/protoboard** works by making electrical connections for you! Each row on each side of the vertical divider of the breadboard is connected together.  The middle divider separates the two halves of the board- the two halves are also not connected together, so for example 1A-1E are connected together or 7F-7J. Wires in the same column are not connected together.
![breadboard](../img/breadBoard.png)
Here's a breadboard diagram (showing yellow rows ar:

	![breadboard diagram](../img/breadboardDiagram.png)

> ### Time to start building Nuvi a house!

Reviewer: I wonder if it makes sense to split this page into two, to make the pages shorter and easier to digest?

Author: sounds good, I'll try that and push the changes one more time

- The **battery pack** is connected to 2 wires (red, black). 
![battery pack with wires](../img/batteryPackWithWires.jpg)
- The **motor** is connected to 2 wires (green, blue). 
![motor with 2 wires](../img/motorWith2Wires.jpg)
- The motor should have an acrylic finger attached.
![motor with acrylic finger](../img/motorWithFinger.jpg)
- The top **toggle switch** is connected to 6 wires (yellow, green, 2 blue, red, black) from its 6 terminals labeled in the figure above. 
It should look something like this:
![switch with wires](../img/switchWithWires.jpg)

Here's a broader view of the part that has the motor, finger and limit switch attached:
![view of piece with motor, finger and limit switch with wires attached](../img/motorFingerAndLimitSwitchPiece.jpg)
- A **breadboard/protoboard** works by making electrical connections for you! Each row on each side of the vertical divider of the breadboard is connected together.  The middle divider separates the two halves of the board- the two halves are also not connected together, so for example 1A-1E are connected together or 7F-7J. Wires in the same column are not connected together.
![breadboard](../img/breadBoard.png)

> Here's a breadboard diagram (showing yellow rows ar:

Reviewer: Typo on this sentence
